operator/v1/register.go

```diff
@@ -54,6 +54,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&KubeSchedulerList{},
 		&KubeStorageVersionMigrator{},
 		&KubeStorageVersionMigratorList{},
+		&MachineConfiguration{},
+		&MachineConfigurationList{},
```

Comment:
Just added MachineConfiguration to operator register.go so @cdoern  could get client-go to generate properly